ssl.c

```diff
@@ -67,10 +67,6 @@ static void ssl_init(void) {
     initialised = 1;
     OpenSSL_add_all_algorithms();
     SSL_library_init();
-
-    /* TODO: Move to sslconnect() */
```

Comment:
This was meant more like "make the context local to the client socket", i.e. move it from global variable to the socket strucuture.